### PR TITLE
docs(samples): included snippet for unmute finding

### PR DESCRIPTION
**Pull request by Sita04**
Added snippet to unmute individual findings.

### CODE REVIEW

File: samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/muteconfig/SetUnmuteFinding.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  // Unmuting a finding that isn't muted has no effect.
+  // Various mute states are: MUTE_UNSPECIFIED/MUTE/UNMUTE.
+  public static void setUnmute(String findingPath) {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+
+      SetMuteRequest setMuteRequest =
+          SetMuteRequest.newBuilder().setName(findingPath).setMute(Mute.UNMUTED).build();
```
Comment (Shabirmean) [inline samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/muteconfig/SetUnmuteFinding.java:52]:
```suggestion
      SetMuteRequest setMuteRequest =
          SetMuteRequest.newBuilder()
              .setName(findingPath)
              .setMute(Mute.UNMUTED)
              .build();
```

File: samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/muteconfig/SetUnmuteFinding.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+      Finding finding = client.setMute(setMuteRequest);
+      System.out.println("Mute value for the finding: " + finding.getMute());
```
Comment (Shabirmean) [inline samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/muteconfig/SetUnmuteFinding.java:55]: Does the "finding" here have a name? If so maybe we can be a bit more descriptive.
`Mute value for the finding " + fiding.name() + " is: " + finding.getMute()`

**Review by Shabirmean — APPROVED** [review]
LGTM!. Left some minor comments

**Comment by Shabirmean** [comment]
I have only reviewed it as a `shadow-reviewer`. We would still need a review from @averikitsch to ensure its still good to go

File: samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/muteconfig/SetUnmuteFinding.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2021 Google LLC
```
Comment (averikitsch) [inline samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/muteconfig/SetUnmuteFinding.java:2]: 2022

**Review by averikitsch — APPROVED** [review]

File: samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/muteconfig/SetUnmuteFinding.java (hunk 1 above)
Comment (Sita04) [reply samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/muteconfig/SetUnmuteFinding.java:52]: The builder syntax was auto-formatted by owlbot to a one-liner.

File: samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/muteconfig/SetUnmuteFinding.java (hunk 2 above)
Comment (Sita04) [reply samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/muteconfig/SetUnmuteFinding.java:55]: Fair enough. Thanks for the suggestion. Modified it.

**Review by catherinerosesantos — APPROVED** [review]

**Review by Neenu1995 — APPROVED** [review]

**Pull request merged**